### PR TITLE
Fix documentation of `sessionDuration`

### PR DESCRIPTION
**Pull request by panicking**
The properties was moved to aws global configuration but the documentation was not updated as it should

Signed-off-by: Michael Trimarchi <michael@amarulasolutions.com>

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by panicking** [comment]
Anyway seems that does not fix my problem. I have increased session time but if the upload time is large the connection drop:

Uploaded 1 artifact(s) to https://amarula-jenkins.s3.eu-west-1.amazonaws.com/novavision/novavision/116/artifacts/
[Pipeline] archiveArtifacts
Archiving artifacts
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)

File: README.md [hunk 1]
```diff
@@ -205,9 +205,9 @@ you cannot download the artifact with that URL any more, thus you have to go bac
 and click on the artifact to download it again. 
 
 If you use a regular Key/Secret AWS Credentials you can set the token duration by adding the property 
-`-Dio.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.sessionDuration` to the Jenkins JVM properties, 
-the default value is one hour. However if you set and IAM Role to assume on the AWS credential, 
-the token duration is always one hour, this parameter does not change the duration in AWS Credentials Plugin settings.
+`-Dio.jenkins.plugins.aws.global_configuration.sessionDuration` to the Jenkins JVM properties,
```
Comment (jglick) [inline README.md:208]:
```suggestion
`-Dio.jenkins.plugins.aws.global_configuration.CredentialsAwsGlobalConfiguration.sessionDuration` to the Jenkins JVM properties,
```
IIUC from https://github.com/jenkinsci/aws-global-configuration-plugin/blame/856a6b6b59e150810181da5823bd7364e19ccab2/src/main/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfiguration.java#L78

File: README.md [hunk 2]
```diff
@@ -205,9 +205,9 @@ you cannot download the artifact with that URL any more, thus you have to go bac
...
+the default value is one hour. However if you set and IAM Role to assume on the AWS credential,
```
Comment (jglick) [inline README.md:209]:
```suggestion
the default value is one hour. However if you set an IAM Role to assume on the AWS credential,
```
while we are here

**Review by jglick — CHANGES_REQUESTED** [review]

**Comment by panicking** [comment]
Done

**Comment by jglick** [comment]
Thanks! Were you able to verify whether this setting indeed fixes your problem?

**Review by jglick — APPROVED** [review]

**Comment by panicking** [comment]
It does not fix it. Works from faster node

[Pipeline] archiveArtifacts
Archiving artifacts
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.io.IOException: java.util.concurrent.TimeoutException
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
[Pipeline] }

Lunch the server with

/usr/bin/java -Djava.awt.headless=true -DsessionTimeout=120 -Djenkins.security.seed.UserSeedProperty.disableUserSeed=true -Xms4096m -Xmx4096m -Djava.util.logging.loglevel=INFO -XX:+DisableExplicitGC -Dio.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.deleteArtifacts=true -Dio.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStoreConfig.deleteStashes=true -Dhudson.DNSMultiCast.disabled=true -Dhudson.udp=-1 -Djava.awt.headless=true -Djenkins.install.runSetupWizard=false -Dhudson.slaves.WorkspaceList=_ -Dio.jenkins.plugins.aws.global_configuration.CredentialsAwsGlobalConfiguration.sessionDuration=7200 -jar /usr/share/java/jenkins.war --webroot=/var/cache/jenkins/war --httpPort=8080 --httpKeepAliveTimeout=60000 --accessLoggerClassName=winstone.accesslog.SimpleAccessLogger --simpleAccessLogger.format=combined --simpleAccessLogger.file=/var/log/jenkins/access_log

[Pipeline] // stage
[Pipeline] mattermostSend
Mattermost Send Pipeline step configured values from global config - connector: false, icon: true, channel: false, color: false[Pipeline] }
[Pipeline] // node
[Pipeline] }
[Pipeline] // throttle
[Pipeline] End of Pipeline
Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to maratona
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1784)
		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)
		at hudson.remoting.Channel.call(Channel.java:1000)
		at hudson.FilePath.act(FilePath.java:1186)
		at hudson.FilePath.act(FilePath.java:1175)

**Comment by jglick** [comment]
Not sure offhand. At any rate, this change does fix the system property name (so far as I can tell), so merging.

**Pull request merged**